planning/obstacle_avoidance_planner/src/mpt_optimizer.cpp

```diff
@@ -411,8 +411,18 @@ void MPTOptimizer::calcPlanningFromEgo(std::vector<ReferencePoint> & ref_points)
 {
   // if plan from ego
   constexpr double epsilon = 1e-04;
-  const bool plan_from_ego =
-    mpt_param_.plan_from_ego && std::abs(current_ego_vel_) < epsilon && ref_points.size() > 1;
+  const double trajectory_length = [&]() {
+    double sum_length = 0;
+    for (size_t i = 1; i < ref_points.size(); ++i) {
+      sum_length +=
+        tier4_autoware_utils::calcDistance2d(ref_points.at(i - 1).p, ref_points.at(i).p);
+    }
+    return sum_length;
+  }();
```

Comment:
can be replaced with tier4_autoware_utils::calcArcLength();
If ReferencePoint msg is not supported in this function, ignore this comment.

Reply:
can use calcArcLength for ReferencePoint so replace it.
https://github.com/autowarefoundation/autoware.universe/pull/1191/commits/3147a55e5a2c447e15db7d0d8be9727cc9fdc47d